frontend/src/scenes/session-recordings/SessionRecordings.tsx

```diff
@@ -212,7 +212,7 @@ function MainPanel(): JSX.Element {
                     <SessionRecordingsPlaylist updateSearchParams />
                 </div>
             ) : tab === ReplayTabs.Playlists ? (
```

Comment:
rename the tab?
i guess we'd have to have a 404 redirect or point playlists and collections to this maybe